src/app/modules/auth/auth.routes.ts

```diff
@@ -1,10 +1,17 @@
 import { celebrate, Joi, Segments } from 'celebrate'
 import { Router } from 'express'
 
+import { limitRate } from '../../utils/limit-rate'
+
 import * as AuthController from './auth.controller'
 
 export const AuthRouter = Router()
 
+/**
+ * @constant {number} SEND_OTP_RATE_LIMIT Per-minute, per-IP limit on /sendotp endpoint
+ */
+const SEND_OTP_RATE_LIMIT = 60
```

Comment:
we should make these optional environment variables that default to a reasonable default

Reply:
I did it this way instead of using env vars because I know we're running into our character limit for env vars on elastic beanstalk. let me figure out exactly how close we are to the limit before I add more env vars - if we get too close to the limit, we might run into production problems e.g. not being able to put up banners when we need to.

Reply:
okay it seems like we still have about 500 characters of space left, so I'll convert these into env vars!